include/deal.II/lac/precondition.h

```diff
@@ -905,8 +959,15 @@ class PreconditionChebyshev : public Subscriptor
 
     /**
      * Stores the inverse of the diagonal of the underlying matrix.
+     *
+     * @deprecated Set the variable @p preconditioner defined below instead.
      */
     VectorType matrix_diagonal_inverse;
```

Comment:
Are you actually still using this variable?

What happens if you mark this variable as `DEAL_II_DEPRECATED`?